src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -83,6 +83,32 @@ string runtime::intelgpu::access_dims(const Shape& dimentions, const AxisSet& ax
     return buffer;
 }
 
+string runtime::intelgpu::reverse_dims(const Shape& dimentions, const AxisSet& axis)
```

Comment:
I see many code duplicated. Maybe it is better to merge this with `access_dims()` with additional parameter with default empty value?